lib/vcmi_endian.h

```diff
@@ -41,14 +48,15 @@ static inline ui32 read_unaligned_u32(const void *p)
 #define read_le_u16(p) (SDL_SwapLE16(read_unaligned_u16(p)))
 #define read_le_u32(p) (SDL_SwapLE32(read_unaligned_u32(p)))
 
-#define PACKED_STRUCT __attribute__(( packed ))
-
 #else
 
+#warning UB: unaligned memory access

```

Comment:
Not sure if there is any point in this "else" block now. Can't think of any compiler that will trigger it